Graphviz/Completions/Attribute Values.sublime-completions

```diff
@@ -1,742 +1,587 @@
 {
-    "scope": "source.dot meta.attributes",
+    "scope": "source.dot & (meta.group | meta.attributes)",
```

Comment:
This kind of selector can cause unwanted results in a world of mixed embedded syntaxes as it also matches `meta.group source.dot` scope.

Order of scopes is important these days. Hence...

```suggestion
    "scope": "source.dot meta.group, source.dot meta.attributes",
```